sqlmesh/core/context.py

```diff
@@ -1031,7 +1031,7 @@ def migrate(self) -> None:
 
         Please contact your SQLMesh administrator before doing this.
         """
-        self._new_state_sync().migrate()
+        self._new_state_sync().migrate(console=self.console)
```

Comment:
why don't you add the console to the state_sync in init?

Reply:
Will update in a sec

Reply:
Done